README.md

```
@@ -1,23 +1,56 @@
+<br>
+
 <p align="center">
-    <img src="docs/images/n8n-surrealdb-icon.png" alt="SurrealDB Icon" width="200">
+    <img width=120 src="https://raw.githubusercontent.com/surrealdb/icons/main/surreal.svg" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)